en/guides/students/login-and-settings/index.md

```diff
@@ -1,199 +1,79 @@
 ---
-title: "[nl] Aanmelden & gebruikersinstellingen"
-description: "Tutorial: aanmelden en gebruikersinstellingen"
+title: "Signing Up & User Settings"
+description: "Tutorial: signing up and user settings"
 order: 2
 ---
 
-::: warning Sorry
-For now, this page is only available in Dutch. Sorry!
-:::
-
-# Aanmelden & gebruikersinstellingen
-> Op deze pagina vind je alle informatie zodat je als student een gebruikersaccount op Dodona kan aanmaken en die kan personaliseren.
 
-**Inhoudsopgave**
-[[toc]]
+# Signing Up & User Settings
+On this page, you will find all the information you need to create and personalize a user account on Dodona as a student.
```

Comment:
```suggestion
> On this page, you will find all the information you need to create and personalize a user account on Dodona as a student.
```

Is there a reason why the table of contents wasn't copied to the English version?

Reply:
Since the latest style change, tables of contest are always include on the right side of the page